src/Raven.Server/Documents/PeriodicBackup/Restore/AbstractRestoreBackupTask.cs

```diff
@@ -106,6 +105,9 @@ public async Task<IOperationResult> Execute(Action<IOperationProgress> onProgres
 
         protected abstract Task RestoreAsync();
 
+        protected virtual void PutSecretKey() => 
```

Comment:
let's have single impl for `PutSecretKey(string databaseName)`

Reply:
done